docs/hoon/library/1c.md

```diff
@@ -9,243 +9,295 @@ navhome: /docs/
 
 Blocksize
 
-Atom representing a blocksize, by convention expressed as a power of 2.
+Atom representing block size. A block of size `a` has a bitwidth of `2^a`.
 
-Source
-------
+#### Source
 
-    ++  bloq  @                                            ::  blockclass
+    ++  bloq  @                                         
 
-Examples
---------
+#### Examples
 
-    > :: ++met measures how many bloqs long an atom is
-    > (met 3 256)
+    ::++met measures how many a-bloqs long an atom is. In other words, how
+    many bloqs of size a would be needed to represent b.
+
+    > (met 3 256)                      ::one 3-bloq has a bidwidth of 8, which can represent 0-255
+    2
+
+    > (met 3 255)                      
+    1
+
+    > (met 3 65.535)                   ::two 3-bloqs together have a bidwidth of 16, which can represent 0-65,535                  
+    > 2              
+
+    > (met 2 65.536)                   
+    > 3
+
+    > (met 0 4)
+    3
+
+    > (met 1 4)
     2
-    > :: 256 is 2 bloqs of 2^3
+
+    > (met 5 9.999.999.999)
+    2
+
+    > (met 2 0xb5)
+    2
+
+#### Discussion
+
+You can think of `bloq` as a numeral system that can represent values of
+`a^2^2`, where `a` is the size of a given block. That's because a block of size
+`a` is `a^2` bits wide, and a binary number that is `b` bits wide can represent
+`b^2` values.
+
```

Comment:
My brain may be a bit fuzzy at the moment, but I find this explanation a little tough to follow.